internal/pkg/scaffold/gopkgtoml.go

```diff
@@ -37,25 +37,9 @@ func (s *GopkgToml) GetInput() (input.Input, error) {
 
 const gopkgTomlTmpl = `# Force dep to vendor the code generators, which aren't imported just used at dev time.
 required = [
-  "k8s.io/code-generator/cmd/deepcopy-gen",
-  "k8s.io/code-generator/cmd/conversion-gen",
-  "k8s.io/code-generator/cmd/client-gen",
-  "k8s.io/code-generator/cmd/lister-gen",
-  "k8s.io/code-generator/cmd/informer-gen",
-  "k8s.io/kube-openapi/cmd/openapi-gen",
-  "k8s.io/gengo/args",
```

Comment:
We can also remove these deps from the `go.mod` scaffold as well, correct?

Reply:
In the `replace` section you mean?